config/locales/es.yml

```diff
@@ -180,7 +180,7 @@ es:
       subtitle_1: railsdevs
       subtitle_2: empodera a los desarrolladores independientes disponibles para su próximo trabajo.
       subtitle_3: Deja de buscar en las bolsas de trabajo y relájate mientras las empresas se comunican contigo.
-      title_1: La bolsa de trabajo invertida para
+      title_1: La bolsa de trabajo enfocada para
```

Comment:
Hmm 🤔 
I left the "invertida" part to keep the fact of railsdevs being a reverse job board. I feel that we'd lose that with "enfocada". What do you think? @joseluistorres 
cc @joemasilotti